### ISSUE TITLE
Add better stylings / components for this simple React app displaying various resources to learn machine learning.

### ISSUE THREAD

**Issue by Parasgupta44** [Parasgupta44/ml-resources-react#3]
### Task
---------
Add some components or improve the existing stylings by doing a PR. Please check `Readme.md` and `Contributing.md` in the repo's root for more details.

**Comment by Tailine** [comment]
Hi @Parasgupta44 , I would like to help on it.

**Comment by Parasgupta44** [comment]
> Hi @Parasgupta44 , I would like to help on it.

Sure, assigned...thanks!

**Comment by CiTRuS93** [comment]
how about using [material-ui's cards](https://material-ui.com/components/cards/)?

**Comment by Parasgupta44** [comment]
> how about using [material-ui's cards](https://material-ui.com/components/cards/)?

Yes, these are pretty awesome...would be great if you can come up with something once you get the time! 
Should I assign this to you as well ?

**Comment by CiTRuS93** [comment]
no I won't steal this from @Tailine , just suggesting an idea 😃

**Comment by Parasgupta44** [comment]
> no I won't steal this from @Tailine , just suggesting an idea 😃

Sure....😄

**Comment by Jawkx** [comment]
I can help, think rounded corner and animation when hover is better. Also, will add a Hero Section

**Comment by Parasgupta44** [comment]
> I can help, think rounded corner and animation when hover is better. Also, will add a Hero Section

Sure...everyone's welcome to contribute. Pretty awesome. 
Just generate a PR and we'll review and merge.

**Comment by Tailine** [comment]
@Parasgupta44 @CiTRuS93 I wasn't thinking about using a component library because the cards are not complicated to do. I will design an example and post here so you guys can take a look.

**Comment by Parasgupta44** [comment]
> @Parasgupta44 @CiTRuS93 I wasn't thinking about using a component library because the cards are not complicated to do. I will design an example and post here so you guys can take a look.

Sure. 
Also, there were some changes in the design. So, you can also check that out from latest `master`

**Comment by CiTRuS93** [comment]
hey, I did some hover -link preview
![hoverpreview](https://user-images.githubusercontent.com/13310488/94805772-94029480-03f5-11eb-9651-2ea471df6b9c.gif)
any thoughts?

**Comment by Parasgupta44** [comment]
> hey, I did some hover -link preview
 
This looks great. Gives a more dynamic feel. (which is awesome!!)
You can gen a PR if you wanna do it...I'll be happy to incorporate it!!

Edit: Also, can you check how responsive the preview panel is for smaller screens ? That'll be great!

**Comment by Tailine** [comment]
> > @Parasgupta44 @CiTRuS93 I wasn't thinking about using a component library because the cards are not complicated to do. I will design an example and post here so you guys can take a look.
> 
> Sure.
> Also, there were some changes in the design. So, you can also check that out from latest `master`

@Parasgupta44 No problem. What do you think of this?

![card](https://user-images.githubusercontent.com/23369123/95003862-3ff4dd00-05ba-11eb-8a33-ec1baff2bb68.png)

**Comment by Jawkx** [comment]
This looks good @Tailine

**Comment by Parasgupta44** [comment]
> @Parasgupta44 No problem. What do you think of this?

This is pretty good. 
Also, I was looking, the page is not that responsive for smaller screens....if that works ,,,it'll be awesome !!

**Comment by Tailine** [comment]
@Jawkx @Parasgupta44 Thanks guys.

**Comment by Tailine** [comment]
@Parasgupta44 Could the responsiveness be another issue?

**Comment by Parasgupta44** [comment]
> @Parasgupta44 Could the responsiveness be another issue?

Sure, done. #38
